spec/lita/clients/harvest_spec.rb

```diff
@@ -1,24 +1,39 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe Lita::Clients::Harvest do
-  let(:client) do 
+  let(:client) do
     described_class.new ENV.fetch('HARVEST_ACCOUNT_ID', 'XXX'), ENV.fetch('HARVEST_TOKEN', 'XXX')
   end
 
-  before do 
-    allow_any_instance_of(described_class).to receive(:to_param).and_return("?to=2017-10-20T23:59:59Z")
+  before do
+    allow_any_instance_of(described_class).to \
+      receive(:to_param).and_return('?to=2018-12-24T23:59:59Z')
   end
 
-  describe "#users_current_entries", vcr: { cassette_name: 'harvest-entries' } do 
-    let!(:entries) { client.users_current_entries }
-    it { expect(entries[3].owner).to eq("Mario López") }
-    it { expect(entries[3].description).to eq("Programming: lita_time_tracker") }
-    it { expect(entries[3].user_email).to eq("mario@platan.us") }
-  end
+  describe '#users_current_entries' do
+    context 'with no inactive users', vcr: { cassette_name: 'harvest-entries-active-users' } do
+      let!(:entries) { client.users_current_entries }
+
+      it 'returns all entries' do
+        expect(entries.count).to eq(5)
+      end
+
+      it { expect(entries.last.owner).to eq('Felipe Domínguez') }
+      it { expect(entries.last.user_email).to eq('felipe.dominguez@platan.us') }
+      it { expect(entries.last.description).to eq('Project Management') }
+    end
+
+    context 'with inactive users', vcr: { cassette_name: 'harvest-entries-inactive-users' } do
+      let!(:entries) { client.users_current_entries }
+
+      it 'returns all entries from active users' do
+        expect(entries.count).to eq(5)
+      end
 
-  describe "#time_entries", vcr: { cassette_name: 'harvest-time_entries' } do 
-    let!(:entry) { client.send("time_entries")[0] }
-    it { expect(entry["spent_date"]).to eq("2017-10-20") }
-    it { expect(entry["started_time"]).not_to eq(nil) } 
+      it 'should not return inactive users_entries' do
```

Comment:
Al escribir los `it` ya no se estila usar los "debería". El tests debe decir lo que debe hacer. Por eso debería ser: `it 'returns inactive...`